about.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="index.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:100,200,300,400,500,600" rel="stylesheet">
+
+</head>
+<body>
+    <div id="navBar">
+        <div id="navLeft">
+            <h1><a href="index.html">Home</a></h1>
+        </div>
+        <div id="navRight">
+            <h3><a href="about.html"> About Me</a></h3>
+            <h3><a href="#recipes"> Recipes</a></h3>
```

Comment:
This link should go to `index.html/#recipes`, right now it only works on the home page